doc/whatsnew/fragments/7264.bugfix

```diff
@@ -0,0 +1,8 @@
+Fixed a case where custom plugins specified by command line could silently fail
+
+Specifically, if a plugin relies on init-hook changing the system path before
```

Comment:
```suggestion
Specifically, if a plugin relies on the ``init-hook`` option changing ``sys.path`` before
```

Reply:
I didn't realise this was an RST file, should have checked. Thanks for the tweaks!